### PR TITLE
Update Readme, Comment out functions in the tutorial, add the php mysql connection test script

### PR DESCRIPTION
**Pull request by lchan-zendesk**
Comment out core logic as described in the tutorial.

(Can move the php script to test db connection to somewhere else.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Example AnyChannel integration with Wordpress using Node Express
+# Example AnyChannel integration with Wordpress using Node Express (Tutorial Version)
 
 ## Description
 This repository contains the source code for an integration service that connects
-Zendesk to Wordpress.  It is intended to be used with the AnyChannel tutorial
-(TODO: Add link once it's available.)
+Zendesk to Wordpress.  It is intended to be used with the AnyChannel tutorial.
+The code referred by the tutorial is commented out in this version. The tutorial has instruction on when to uncomment the commented out code.
```
Comment (smedberg) [inline README.md]: nit: "The code referred by the tutorial is commented out in this version. The tutorial has instruction on when to uncomment the commented out code." => "The tutorial instructs you to implement functionality in the wordpress.js file. Relevant code is already present in wordpress.js, but is commented out." or something like that?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-# Example AnyChannel integration with Wordpress using Node Express
+# Example AnyChannel integration with Wordpress using Node Express (Tutorial Version)
```
Comment (smedberg) [inline README.md:1]: nit: if you're pointing out that this is the tutorial version, is it worth mentioning where the non-tutorial version is?

**Comment by smedberg** [comment]
:+1:

**Comment by lchan-zendesk** [comment]
@smedberg Added an extra line `NOTE: The non-tutorial version is located at https://github.com/zendesk/sample_wordpress_anychannel_integration` to the Readme.

**Comment by smedberg** [comment]
:+1:

**Pull request merged**